config/1-redirects.yml

```diff
@@ -0,0 +1,54 @@
+INHERIT: 0-base.yml
```

Comment:
All redirects are defined in this file, adding a redirect should be as simple as adding a new key-value pair